specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProperty.json

```diff
@@ -65,182 +95,553 @@
+          "enum": [
+            "Unknown",
+            "Active",
+            "Deleted",
+            "Disabled",
+            "UnderReview",
+            "Warned",
+            "Restricted"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "string"
+          "x-ms-enum": {
+            "name": "InvoiceSectionStatus",
+            "modelAsString": true
+          }
+        },
+        "invoiceSectionStatusReasonCode": {
+          "description": "Reason for the specified invoice section status.",
+          "enum": [
+            "PastDue",
+            "UnusualActivity"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "InvoiceSectionStatusReasonCode",
+            "modelAsString": true
+          }
+        },
+        "isTransitionedBillingAccount": {
+          "description": "Specifies if the billing account for the subscription is transitioned from a Microsoft Online Service Program (MOSP) to a Microsoft Customer Agreement (MCA) account. Will be present and value will be true if its a transitioned billing account.",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "skuDescription": {
+          "description": "The sku description.",
+          "type": "string",
+          "readOnly": true
         },
-        "isAccountAdmin": {
-          "description": "Indicates whether user is the account admin.",
+        "skuId": {
+          "description": "The ID that uniquely identifies a sku.",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionBillingStatus": {
+          "description": "The subscription status.",
+          "enum": [
+            "Unknown",
+            "Active",
+            "Disabled",
+            "Deleted",
+            "Warned",
+            "Expiring",
+            "Expired",
+            "AutoRenew",
+            "Cancelled",
+            "Suspended"
+          ],
+          "type": "string",
           "readOnly": true,
-          "type": "boolean"
+          "x-ms-enum": {
+            "name": "BillingSubscriptionStatus",
+            "modelAsString": true
+          }
         },
-        "productId": {
-          "description": "The product ID of the Azure plan.",
+        "subscriptionBillingStatusDetails": {
+          "description": "The reason codes for the subscription status.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BillingSubscriptionStatusDetails"
+          },
           "readOnly": true,
-          "type": "string"
+          "x-ms-identifiers": []
         },
-        "productName": {
-          "description": "The product name of the Azure plan.",
+        "subscriptionBillingType": {
+          "description": "The type of billing subscription.",
+          "enum": [
+            "None",
+            "Benefit",
+            "Free",
+            "Paid",
+            "PrePaid"
```

Comment:
Why do we need "Prepaid"? This value isn't really meaningful or accurate.

Reply:
Also, what does Benefit mean as a billing type? We should reconcile this also with productCategory...

Reply:
@amberbhargava @ramkd12 to comment